src/translations/nbTranslations.json

```diff
@@ -1175,7 +1175,7 @@
       "confirm_remove_vocabulary_title": "Fjern vokabular?",
       "create_project_helper_text": "<0>Forskeren definerer selv sine prosjekter.</0> <0>Et prosjekt må ha en tittel, en prosjektleder, en koordinerende institusjon, en start- og sluttdato. Et prosjekt kan ha prosjektdeltakere, finansiørerer, godkjenninger eller forskningsresultat.</0> <0>Om du har en NFR finansiering så kan du etablere et prosjekt med den som utgangspunkt. Da starter du med å søker du opp din NFR finansiering, den blir lagt til prosjektet og alle obligatoriske felt er ferdig utfylt, foruten prosjektleder som må søkes opp og bekreftes.</0> <0>(Det planlegges tilsvarende funksjonalitet med utgangspunkt i en REK-godkjenning. Alle prosjekter som skal være del av KBS-rapporteringen må ha en REK-godkjenning som identifiserer prosjektet som en klinisk behandlingsstudie.)</0>",
       "date_published": "Publiseringsdato",
-      "description_of_content": "Beskrivelse av innhold, hvis relevant",
+      "description_of_content": "Beskrivelse av innhold, spesielt viktig for blant annet kunstnerisk resultat",
```

Comment:
FYI, mener at dette vil vises som en heading på landing page for alle publiseringer som _ikke_ er kunstnerisk resultat også, så det kommer sikkert en ny task på å rette opp det i etterkant om vi ruller ut dette. Om det ikke for en merkelig grunn er ønsket da 😅

Reply:
Skal dobbeltsjekke at denne ikke blir gjenbrukt flere plasser.

Reply:
Ser ut til at den gjenbrukes i sentralimport registreringspanelet. Det er vel greit.

Reply:
Ah, jeg var helt sikker på at denne ble brukt på landing page også, men ser at det ikke stemmer nå ja!